playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -0,0 +1,87 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Copy skuba openstack terraform files to socok8s workspace
+  copy:
+    src: "{{ playbook_dir }}/../submodules/skuba/ci/infra/openstack/"
+    dest: "{{ socok8s_workspace }}/skuba-terraform/"
+    mode: 0775
+    force: no
+
+- name: Create a terraform variable file
+  template:
+    src: skuba-terraform.tfvars.j2
+    dest: "{{ socok8s_workspace }}/skuba-terraform/terraform.tfvars"
+  vars:
+    # TODO(toabctl): For now just use the image that the CaaSP team is using
+    image_name: "SLE-15-SP1-JeOS-GM"
```

Comment:
I would prefer using this as a defaults/main.yml variable, so that people can override it if they want.

Reply:
Done